tests/testthat/test-subset-access.R

```diff
@@ -187,15 +187,29 @@ test_that("Duplicate colvars path correctly", {
                      res)
 })
 
-test_that("top_left retention behavior is correct across all scenarios", {
+test_that("top_left, title, footers retention behaviors are correct across all scenarios", {
```

Comment:
add reindex_refs here too so that all the basic checks for `[` behavior are in one place